GOVERNANCE.md

```diff
@@ -0,0 +1,93 @@
+# cert-manager Governance
+
+This document defines project governance for the cert-manager project.
+
+## Contributors
+
+cert-manager is for everyone. Anyone can become a cert-manager contributor
+simply by contributing to the project, whether through code, documentation, blog
+posts, community management, or other means. As with all cert-manager community
+members, contributors are expected to follow the [cert-manager Code of
+Conduct][coc].
+
+All contributions to cert-manager code, documentation, or other components in
+the cert-manager GitHub org must follow the guidelines in [the contributing
+page][contrib]. Whether these contributions are merged into the project is the
+prerogative of the maintainers.
+
+## Maintainers
```

Comment:
maintainer
= someone who can correspond with CNCF on behalf of the project
= someone who can participate in a "maintainers vote"